t/lib_with_a+/TestsFor/DateTime.pm

```diff
@@ -0,0 +1,21 @@
+# straight from TCM POD
+package TestsFor::DateTime;
+use Test::Class::Moose;
+use DateTime;
```

Comment:
This would add DateTime as a test prereq, which isn't really necessary.

I think the test can just do `ok(1, 'test ran')` and that's good enough.